### PR TITLE
Pileup style tweaks in Cycledash.

### PR DESCRIPTION
**Pull request by jaclynperrone**
- Updates css to accommodate pileup 0.2 release
- Replaces previous, next, zoom and close buttons with glyphicons

screen shot 2015-07-06 at 11 54 59 am

[Review on Reviewable](https://reviewable.io/reviews/hammerlab/cycledash/777)

### CODE REVIEW

**Comment by jaclynperrone** [comment]
Looping @danvk in. Would love some feedback from you as well!

**Comment by danvk** [comment]
It's pileup 0.2, not 2.0 :)

**Comment by ihodes** [comment]
This looks splendid, thank you. Do the arrow keys/esc still work? (Did they ever work? I may be making things up.)

File: cycledash/static/scss/examine.scss [hunk 1]
```diff
@@ -549,7 +584,7 @@ a.download-vcf {
 }
 
 #pileup-container .controls {
-  flex: 0 0 35px;  /* fixed height */
+  flex: 0 0 38px;  /* fixed height */
```
Comment (danvk) [inline cycledash/static/scss/examine.scss]: The added height is to accommodate the larger zoom in/out buttons?

**Comment by danvk** [comment]
@ihodes they used to but they don't now because I've punted on a notion of whether the pileup viewer has focus. Feel free to file an issue to add them back.

@jaclynperrone This looks great. One question is whether the big prev/next buttons should be up/down or left/right. You may or may not be moving backward/forward in the genome (if the variant list is sorted by position). You're definitely moving up/down in the list. I could go either way on this, up/down buttons might feel a bit odd in that position.

Otherwise looks great!

**Comment by ihodes** [comment]
@dan https://github.com/hammerlab/cycledash/issues/696 https://github.com/hammerlab/cycledash/issues/778

File: cycledash/static/scss/examine.scss (hunk 1 above)
Comment (jaclynperrone) [inline cycledash/static/scss/examine.scss]: Yup! But I actually don't need that much height anymore. Gonna push a tiny tweak.

**Comment by jaclynperrone** [comment]
@danvk I would definitely stick with the left/right arrows. I think the up/down would be confusing. I just pushed a tiny update to the control panel height.

@ihodes Thanks for taking a look! Glad you like it :)

**Comment by danvk** [comment]
LGTM

**Pull request merged**